### PR TITLE
chore: Small tweaks

### PR DESCRIPTION
**Pull request by jacobdalamb**

### CODE REVIEW

File: astro.config.mjs
Comment (mre) [inline astro.config.mjs:1]: Can the indentation be configured? I think two spaces would be more readable.

**Comment by mre** [comment]
Looks good. Added one comment and you need to rebase on top of `master`.

File: astro.config.mjs
Comment (jacobdalamb) [reply astro.config.mjs:1]: Yes.

**Comment by mre** [comment]
Very nice! ⭐

**Pull request merged**